gpbft/gpbft.go

```diff
@@ -567,26 +630,11 @@ func (i *instance) tryQuality() error {
 	return nil
 }
 
-func (i *instance) beginConverge() {
+// beginConverge initiates CONVERGE_PHASE justified by the given justification.
+func (i *instance) beginConverge(justification *Justification) {
 	i.phase = CONVERGE_PHASE
-
 	i.phaseTimeout = i.alarmAfterSynchrony()
```

Comment:
Suggestion. Add an assertion that the justification is for `i.round - 1`. I think we can panic if not.